packages/core/app-layout/src/components/sidebar/AppSidebar.vue

```diff
@@ -454,23 +454,23 @@ onBeforeUnmount(() => {
 
 .sidebar-top {
   align-items: center;
-  color: #fff;
+  color: $kui-color-background;
   display: flex;
-  margin: 0 0 16px;
-  padding: 0 8px;
+  margin: $kui-space-0 $kui-space-0 $kui-space-60;
+  padding: $kui-space-0 $kui-space-40;
   user-select: none;
 }
 
 .sidebar-item-external-link {
   align-items: center;
   display: flex;
-  font-size: 14px;
+  font-size: $kui-font-size-30;
   justify-content: space-between;
-  line-height: 1.3;
+  line-height: $kui-line-height-30;
   text-decoration: none;
 
   &:focus-visible {
-    outline: 1px solid var(--steel-300, #A3B6D9) !important;
+    outline: 1px solid $kui-color-text-neutral-weak !important;
```

Comment:
Did you mean to use `color-text` token for outline?

Reply:
Fixed